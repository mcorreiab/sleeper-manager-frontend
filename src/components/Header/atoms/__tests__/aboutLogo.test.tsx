import renderer from "react-test-renderer";
import AboutLogo from "../aboutLogo";

it("Should render the svg about logo", () => {
  const tree = renderer.create(<AboutLogo />).toJSON();
  expect(tree).toMatchSnapshot();
});
