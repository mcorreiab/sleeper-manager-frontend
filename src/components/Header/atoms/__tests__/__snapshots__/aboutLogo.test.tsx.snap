// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the svg about logo 1`] = `
<svg
  aria-labelledby="projectInfoIconId"
  fill="none"
  height="32"
  role="img"
  viewBox="0 0 32 32"
  width="32"
  xmlns="http://www.w3.org/2000/svg"
>
  <title
    id="projectInfoIconId"
  >
    More information about sleeper manager
  </title>
  <path
    clipRule="evenodd"
    d="M16 0C7.168 0 0 7.168 0 16C0 24.832 7.168 32 16 32C24.832 32 32 24.832 32 16C32 7.168 24.832 0 16 0ZM16 24C15.12 24 14.4 23.28 14.4 22.4V16C14.4 15.12 15.12 14.4 16 14.4C16.88 14.4 17.6 15.12 17.6 16V22.4C17.6 23.28 16.88 24 16 24ZM14.4 11.2001H17.6V8.0001H14.4V11.2001Z"
    fill="#DADADA"
    fillRule="evenodd"
  />
</svg>
`;
